src/components/ExpandableCard.js

```diff
@@ -50,11 +50,21 @@ const Question = styled.div`
   margin-right: 1rem;
 `
 
+const Header = styled.div`
+  display: flex;
+  width: 100%;
+  margin: 1rem 0;
+  align-items: center;
+  img {
+    margin-right: 1.5rem;
+  }
+`
+
 const ButtonContainer = styled.div`
   margin-left: 1rem;
 `
 
-const ExpandableCard = ({ children, contentPreview, title }) => {
+const ExpandableCard = ({ children, contentPreview, title, svg }) => {
```

Comment:
Not a fan of how we have to click directly on the 'More' text to expand the additional content - [example GIF](https://gyazo.com/574e7a8b9b8dd4b9a0cf7712908574cc). This is quite a small target and [could be difficult for mobile users to interact with](https://www.nngroup.com/articles/touch-target-size/).

Reply:
I think it'd be an improvement if we make the entire Card clickable - [example GIF](https://gyazo.com/8f4f0e9f028d08572dd5048df9db3bb4). 

I've checked and we also use this on the History page and the [staking community grants page](https://ethereum.org/en/community/grants/staking-community-grants), neither are negatively affected by the change.

Reply:
Yeah, personally prefer being able to click the entire card as well. Tried to start by reusing components, but would be down to modify this.

Reply:
I tend to agree but think we can keep this as a separate PR, given that we use this card component all over the place.

Reply:
@wackerow please create a separate story in this epic to address this.